include/fi_util.h

```diff
@@ -418,23 +433,26 @@ struct util_cmap {
 
 	struct ofi_key_idx key_idx;
 
+	uint8_t closing;
```

Comment:
This variable is used to signal that the whole of cmap is about to be closed / freed. The cmap_state is for individual handles / connection in the cmap.